Cargo.toml

```diff
@@ -24,6 +24,7 @@ parking_lot = "0.12"
 prometheus-client-derive-encode = { version = "0.4.1", path = "derive-encode" }
 prost = { version = "0.11.0", optional = true }
 prost-types = { version = "0.11.0", optional = true }
+quantiles = "0.7.1"
```

Comment:
I would guess that most users of this library don't need the `Summary` type. What do you think of feature flagging the type as well as the `quantiles` dependency. The latter will help keeping ones dependency tree lean, thus compile times short.

Reply:
Are we blocked by this issue?